src/parser/statement.js

```diff
@@ -844,7 +844,7 @@ pp.parseExport = function (node) {
     }
     node.declaration = expr;
     if (needsSemi) this.semicolon();
-    this.checkExport(node);
+    this.checkExport(node, true, true);
```

Comment:
just a thought: might be nice if this method wasn't taking in true, true (an object, named, etc) but I know we do that everywhere else ^ `this.parseFunction(expr, true, false, false, true);` 😄